_vector-search/performance-tuning-search.md

```diff
@@ -45,5 +45,46 @@ This API operation only loads the segments of active indexes into the cache. If
 
 ## Avoid reading stored fields
 
-If your use case only involves reading the IDs and scores of the nearest neighbors, you can disable the reading of stored fields, which saves time that would otherwise be spent retrieving the vectors from stored fields.
+If your use case only involves reading the IDs and scores of the nearest neighbors, you can disable the reading of stored fields, which saves time that would otherwise be spent retrieving the vectors from stored fields. To disable stored fields entirely, set `_source` to `false`:
 
+```json
+GET /my-index/_search
+{
+  "_source": false,
+  "query": {
+    "knn": {
+      "vector_field": {
+        "vector": [ 0.1, 0.2, 0.3],
+        "k": 10
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+This query returns only the document IDs and scores, making it the fastest option when you don't need the actual document contents. For more information, see [Disabling `_source`]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/retrieve-specific-fields/#disabling-_source).
+
+## Exclude vectors from search results
+
+If you need the document contents but want to optimize performance, you can exclude just the vector fields from being returned in the search results. This approach reduces network transfer while still maintaining access to other document fields. To exclude vectors from search results, provide the vector field name in `_source.excludes`:
```

Comment:
could you link to above search-plugins/searching-data/retrieve-specific-fields to learn more as well? theres some nuance in the most optimal way and itd be good to point to the overall docs for sake of completion

Reply:
Sure, done!